Makefile

```diff
@@ -9,6 +9,23 @@ appendonly no
 requirepass passwd
 endef
 
+# STANDALONE REDIS NODE WITH SSL
```

Comment:
Travis tests are not running in this branch for some reason.

When I tried to run it locally I'm getting the following

```
Oleksiis-MacBook-Pro:Projects oleksiidiagiliev$ cd spark-redis
Oleksiis-MacBook-Pro:spark-redis oleksiidiagiliev$ make start
make start-standalone
echo "$REDIS_STANDALONE_NODE_CONF" | redis-server -
echo "$REDIS_STANDALONE_NODE_CONF_SSL" | redis-server -

*** FATAL CONFIG FILE ERROR (Redis 6.0.1) ***
Reading the configuration file, at line 8
>>> 'tls-auth-clients no'
Bad directive or wrong number of arguments
make[1]: *** [start-standalone] Error 1
make: *** [start] Error 2
```